packages/search-ui-app-search-connector/src/responseAdapter.js

```diff
@@ -2,10 +2,11 @@ function adaptation1AdaptFacetValue(
   facetValue,
   additionalFacetValueFieldsForField = {}
 ) {
+  const hasValue = facetValue.hasOwnProperty("value");
   const { count, value, ...rest } = facetValue;
   return {
     count,
-    value: value
+    value: hasValue
```

Comment:
This is the main fix for the bug. If I were to do that, and a `null` value was returned, then the second part of that ternary would be returned, which would be an empty object (`{}`), and it would be broken anyway.

Reply:
Makes total sense now, thanks for the clarification.